docs/application/web/guides/multimedia/metadata.md

```diff
@@ -2,34 +2,41 @@
 
 You can extract metadata information from multimedia files.
 
-The main features of the Metadata API include:
-
-- Extraction of a metadata from a file
-  You can [extract metadata](#extract-metadata) from multimedia files using `get()` method.
+The main feature of the Metadata API is extraction of a metadata from a file. You can [extract metadata](#extract-metadata) from multimedia files using the `get()` method.
 
 
 ## Prerequisites
 
-To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html)), the application has to request [proper storage permissions](../security/privacy-related-permissions.md).
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html) and [tv](../../api/latest/device_api/tv/tizen/metadata.html) applications), the application has to request storage privileges by adding the following privileges to the `config.xml` file:
```

Comment:
KIndly add "," after **and** (Serial comma rule)
```suggestion 
To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html), and [tv](../../api/latest/device_api/tv/tizen/metadata.html) applications), the application has to request storage privileges by adding the following privileges to the `config.xml` file:

Reply:
@pkosko  Kindly check the links. If Ok , ignore the comment.
@mijin85cho I presume that the page is new one, two links that are not working will work

Reply:
All links work well on staging page generated automatically:
https://docs1.stg.tizen.org/staging/1175/application/web/guides/multimedia/metadata/

Reply:
@pkosko So sorry. you missed this

Kindly add "," after and (Serial comma rule)

https://github.com/Samsung/tizen-docs/pull/1175#discussion_r454187379